prql-compiler/src/test.rs

```diff
@@ -2954,6 +2954,20 @@ fn test_errors() {
        ·                       ┬
        ·                       ╰── unexpected ’
     ───╯
+    Error:
+       ╭─[:1:36]
+       │
+     1 │ Mississippi has four S’s and four I’s.
+       ·                                    ┬
+       ·                                    ╰── unexpected ’
+    ───╯
+    Error:
+       ╭─[:1:39]
+       │
+     1 │ Mississippi has four S’s and four I’s.
+       ·                                       ┬
+       ·                                       ╰── identifier expected an identifier or *, but found
```

Comment:
It's not great prose, but we can iterate...

Reply:
I did not know that at the time, but expected token `None` means end of file. In the convert_parser_error I'm converting that `None` into empty string, which causes this output.

Reply:
Great, fixed in #2008, thanks